pkg/virt-controller/watch/migration_test.go

```diff
@@ -584,6 +599,43 @@ var _ = Describe("Migration watcher", func() {
 			shouldExpectMigrationSchedulingState(migration)
 			controller.Execute()
 		})
+
+		table.DescribeTable("should handle pod stuck in unschedulable state", func(phase virtv1.VirtualMachineInstanceMigrationPhase, shouldTimeout bool, timeLapse int64) {
```

Comment:
I'm not fond of defining the expected test state in the test's input parameters - i.e. the `shouldTimeout` boolean input argument - but I don't see simpler alternatives.

Reply:
this is a fairly common pattern we use with tables. Pass in various input and declare what output is expected. It's a way to condense more tests into a single table.